client/fidelity/compliance-review/views/compliance-review.html

```diff
@@ -5,30 +5,34 @@
                 {{ search.labelTo }}
                 <span ng-if="items._meta.total">
                     (<span>{{ items._meta.total }}</span> <span>{{
-                            (items._meta.total === 1 ? 'item' : 'items') | translate
-                    }}</span>)
+                        (items._meta.total === 1 ? 'item' : 'items') | translate
```

Comment:
`<div translate translate-n="boatCount" translate-plural="{{$count}} boats">One boat</div>` this is how plural needs to be translated. https://angular-gettext.rocketeer.be/dev-guide/annotate/

Reply:
@tomaskikutis cool, did we have this before? Because that code was already in another PR; I can change it on this one tho

Reply:
Yep, we had this all along :smile:

Reply:
Okay, I thought https://github.com/superdesk/superdesk-client-core/pull/2818 was introducing plurals but I guess it's for gettext only

Reply:
That PR is only adding a wrapper so it can be used from React without having to use `connectServices`.